Sources/TuistSupport/Xcode/XcodeController.swift

```diff
@@ -21,17 +21,24 @@ public class XcodeController: XcodeControlling {
     /// Shared instance.
     public static var shared: XcodeControlling = XcodeController()
 
+    /// Cached response of `xcode-select` command
+    private var selectedXcode: Xcode?
```

Comment:
Would you mind making the access to this variable atomic? This variable is not thread safe and it might cause race conditions and crashes. I think it's a good use case for the [Atomic](https://www.onswiftwings.com/posts/atomic-property-wrapper/) property wrapper. What do you think?

Reply:
Good idea, I did not think about thread safety 👍